sormas-ui/src/main/java/de/symeda/sormas/ui/contact/ContactsView.java

```diff
@@ -377,6 +380,21 @@ public VerticalLayout createFilterBar() {
 		});
 		filterLayout.addComponent(filterForm);
 
+		HorizontalLayout actionButtonsLayout = new HorizontalLayout();
+		actionButtonsLayout.setSpacing(true);
+		{
+			// Follow-up overview scrolling
+			if (ContactsViewType.FOLLOW_UP_VISITS_OVERVIEW.equals(viewConfiguration.getViewType())) {
+				filterLayout.setWidth(100, Unit.PERCENTAGE);
+				HorizontalLayout scrollLayout = buildScrollLayout();
```

Comment:
this name is still misleading, no scroll layout is being built there, please change it

Reply:
the method name has been changed both on cases and contacts